src/core/prompts/system-prompt/tools/list_code_definition_names.ts

```diff
@@ -15,7 +15,7 @@ const generic: ClineToolSpec = {
 		{
 			name: "path",
 			required: true,
-			instruction: `The path of the directory (relative to the current working directory {{CWD}}) to list top level source code definitions for.`,
+			instruction: `The path of the directory (relative to the current working directory {{CWD}}){{MULTI_ROOT_HINT}} to list top level source code definitions for.`,
```

Comment:
```suggestion
			instruction: `The path of the directory (relative to the current working directory {{CWD}}) to list top level source code definitions for.`,
```
If MULTI_ROOT_HINT always follow {{CWD}}  can't we just use the same {{CWD}} placeholder with Multi root hint appended?

Reply:
<img width="685" height="433" alt="image" src="https://github.com/user-attachments/assets/e21e8615-1e9a-441d-9389-4c36dd79a087" />
Structurally this makes a lot of sense as its clear differentiation and also supports very clean backwards compatibility.